cmd/crictl/attach.go

```diff
@@ -50,6 +50,24 @@ var runtimeAttachCommand = &cli.Command{
 			Value:   transportSpdy,
 			Usage:   fmt.Sprintf("Transport protocol to use, one of: %s|%s", transportSpdy, transportWebsocket),
 		},
+		&cli.StringFlag{
+			Name:    flagTLSServerName,
+			Usage:   "Server name used in the TLS client to check server certificates against",
+			Aliases: []string{"tls-server", "tls-sn"},
```

Comment:
should it be `tls-sni` like https://en.wikipedia.org/wiki/Server_Name_Indication?

Reply:
Yes, changed as suggested and kept the `tls-server-name` alias.